autoscheduler/scraper/views.py

```diff
@@ -1,6 +1,13 @@
+from collections import OrderedDict
 from rest_framework import generics
+from rest_framework.response import Response
 from .serializers import CourseSerializer
+from .serializers import SectionSerializer
+from .serializers import TermSerializer
+from .serializers import CourseSearchSerializer

```

Comment:
```suggestion
from scraper.serializers import CourseSerializer
```

Ryan and I decided that we should use "exact" imports rather than relative, so apply this to the rest of the serializer imports

Reply:
In addition, we should format it like this so we only need to use one import statement:
```suggestion
from scraper.serializers import (
    CourseSerializer, CourseSearchSerializer, SectionSerializer, TermSerializer
)
```